grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -84,7 +84,7 @@ class BulkDownloadService {
 
             zipStream.putNextEntry new ZipEntry(filenameToUse)
 
-            def bytesCopied = IOUtils.copy(streamFromUrl, zipStream)
```

Comment:
Wanna help me to see how we can pull it down? I had a bunch of conflicts because grails /sort of/ implements it. So we can pull apache.commons.io, but we're using just this little function. Can be changed.